src/components/Toolbar/index.jsx

```diff
@@ -109,11 +109,11 @@ class Toolbar extends React.Component {
           </Fragment>
         )}
         {children}
-        {!noRefreshBtn && (
+        {/* {!noRefreshBtn && (
```

Comment:
why comment this block ?

Reply:
New design has not refresh button in table